### ISSUE TITLE
Questions regarding Detector 

### ISSUE THREAD

**Issue by Sentient07** [phil-bergmann/tracking_wo_bnw#96]
Hello,

I have a couple of questions regarding the detector network. 

1) In the paper you seemed to have mentioned the detection backbone as `Resnet-101`, however in the code, the class `FRCNN_FPN` seems to use a Resnet-50 backbone. Could you please elaborate if you've used `Resnet-101` in your experiments? If so, can you also please clarify whether there were improvements in MOTA scores with a `Resnet-101` backbone(for private detection experiments)

2) The `FRCNN_FPN` class inherits `FasterRCNN` class which imposes a [limitation](https://github.com/pytorch/vision/blob/v0.3.0/torchvision/models/detection/faster_rcnn.py#L155) on the number of objects to detect to 100. For private detection experiments, I think this constraint needs to be lifted as in the MOT-20 sequences, there are 200+ persons in frames. I have some improvements in MOTA score making this change remotely. If you think it's a good option to have that change for the Private detection mode, I will be happy to send a Pull Request.

Thanks,

**Comment by Sentient07** [comment]
(contd..)
3) Also, I would be much grateful if you can also release the weights for detector, trained on the training sequences of the MOT-19/20 benchmarks. Thank you!

**Comment by bluesky314** [comment]
@phil-bergmann Any update on this?

**Comment by Sentient07** [comment]
I hope this helps,
Link : https://drive.google.com/file/d/1g8fwFF17B6yk6yLxr5y-hxRsGLCyZsbl/view?usp=sharing


![Tracktor_accuracy](https://user-images.githubusercontent.com/6195312/90333909-fa5c6080-dfc9-11ea-847c-a6f835549d25.png)

**Comment by timmeinhardt** [comment]
1. We used ResNet101 in the paper but then updated the code to a newer PyTorch version, retrained the model file and achieved a performance boost with ResNet50. Probably more related to a better training schedule and not only the smaller backbone. The repository includes the old (ResNet101) model files and code in a separate `iccv_19` branch.
2. Your concern is valid but only for private detections mode. For public detections we use our custom `predict_boxes` method which does not have this limitation. I will increase the maximum number in our code. Thank you for pointing this out.
3. We uploaded results and model files for MOT20.
